spec/readthis/cache_spec.rb

```diff
@@ -192,6 +197,11 @@
       expect(cache.read('b')).to be_nil
       expect(cache.read('b', namespace: 'alph')).not_to be_nil
     end
+
+    it 'return empty results without keys' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.